network-api/networkapi/wagtailpages/tests/test_blog.py

```diff
@@ -22,10 +22,11 @@ class BlogIndexTestCase(test_base.WagtailpagesTestCase):
     def setUpTestData(cls):
         super().setUpTestData()
         cls.page_size = blog_index.BlogIndexPage.PAGE_SIZES[0][0]
-        cls.blog_index = blog_factories.BlogIndexPageFactory(
+        blog_factories.BlogIndexPageFactory(
             parent=cls.homepage,
             page_size=cls.page_size,
         )
+        cls.blog_index = blog_models.BlogIndexPage.objects.first()
```

Comment:
This was causing all the blog tests to fail. I can't quite describe why, I assume it's something to do with targeting the method and not the object.

Reply:
Very curious.